Vagrantfile

```diff
@@ -124,7 +124,7 @@ cat << EOF > /home/vagrant/.ipython/kernels/toree-kernel/kernel.json
     "display_name": "Spark 1.5.1 (Scala 2.10.4)",
```

Comment:
I removed the version all together to not cause any confusion between the version of Toree and Spark. To know what version of Spark a Toree build is compatible with, users should reference our site.